### DISCUSSION TITLE
Max width, like Max height

### DISCUSSION THREAD

**Discussion by gullevek** [rxhanson/Rectangle#509] [Ideas]
I was surprised that there is no max width like there is a max height.

Is there a reason for it not existing?

**Comment by rxhanson** [comment]
I guess it comes down to that personally I never feel a need to maximize the width of a window, in part because I don't use portrait orientation displays. If I remember correctly, maximize height might have been a user request for Spectacle that I liked and decided to add in early on in Rectangle. Maximizing Height is also done on Windows if you resize a window to a top or bottom edge, and that also made it seem like a command that other users would find useful.

In terms of adding it in, I would prefer to only add it to the UI if there was a lot of user demand for it. There's not a way to add this in via terminal command at the moment, although the `SpecifiedCalculation` could be modified to allow this behavior (https://github.com/rxhanson/Rectangle#add-an-extra-centering-command-with-custom-size). If you would like to create a PR with this, we can discuss further.

As an aside, it is simple to create a custom command for this in my paid superset of Rectangle, [Hookshot](https://hookshot.app).

**Reply by gullevek** [reply]
Thanks. I am the only one and I also have BTT where I have that setup anyway. So unless a million people come forth and want that too it's not needed. I was just wondering why one existed and not the other.
